spark/1.11/README.md

```diff
@@ -0,0 +1,160 @@
+
+- Estimated time for completion: 10 minutes
+- Target audience: Anyone interested in a data processing engine.
+- Scope: Learn how to install Spark on DC/OS and how to launch a Spark job. 
+
+**Terminology**:
+
+- **Application** ... A program built on Spark, consisting of a driver program and executors in the cluster.
+- **Driver** ... A process running the `main()` function of the application (and providing the SparkContext).
+- **Executor** ... A process launched for an application on a worker node.
+- **Mesos run modes** ... Spark can run in Mesos using two modes: `coarse-grained` (default) and `fine-grained`. The `coarse-grained` mode has a lower startup overhead and is good for interactive sessions; it launches one long-running Spark task on each worked node at the cost of reserving the resources for the complete duration of the application. The `fine-grained` mode shows better resource utilization and is good for batch and certain streaming jobs; it launches each Spark task as a separate Mesos task. Note that `fine-grained` mode was [deprecated](https://spark.apache.org/docs/latest/running-on-mesos.html#fine-grained-deprecated) as of Spark 2.0.0.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Spark](#install-spark)
+- [Launch a Spark job](#launch-a-spark-job)
+- [Uninstall Spark](#uninstall-spark)
+
+## Prerequisites
+
+- A running DC/OS 1.11 cluster with at least 3 nodes with each 2 CPUs and 2 GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.11/cli/install/) installed.
+
+## Install Spark
+
+Assuming you have a DC/OS cluster up and running, the first step is to install Spark:
+
+```bash
+$ dcos package install spark
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#certified-services
+Continue installing? [yes/no] yes
+Installing Marathon app for package [spark] version [2.3.1-2.2.1-2]
+Installing CLI subcommand for package [spark] version [2.3.1-2.2.1-2]
+New command available: dcos spark
+DC/OS Spark is being installed!
+
+        Documentation: https://docs.mesosphere.com/service-docs/spark/
+        Issues: https://docs.mesosphere.com/support/
+```
+
+If the spark service is already installed and you only wish to install the CLI subcommand on your client, simply do:
+
+```bash
+$ dcos package install spark --cli
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#certified-services
+Continue installing? [yes/no] yes
+Installing CLI subcommand for package [spark] version [2.3.1-2.2.1-2]
+New command available: dcos spark
+```
+
+Note that while the DC/OS CLI subcommand `spark` is immediately available, it takes a few moments for Spark to start running in the cluster.
+
+To see the new possible with the new `spark` subcommand, you can use the following command:
+
+```bash
+$dcos spark --help
```

Comment:
Since we're making a couple edits, can you add a space between the $ and dcos like the others have?